specs.md

```diff
@@ -209,7 +225,7 @@ BMS-20 | Exiting the drive state and entering the fault state | When an AIR shut
 ID | Title | Description | Associated Competition Rule(s)
 --- | --- | --- | ---
 BMS-29 | Entering the fault state | The BMS must open the AIR+. 
-BMS-30 | Exiting the fault state and entering the init state | Once all AIR shutdown faults are cleared, the BMS must exit the fault state and enter the init state.
+BMS-30 | Exiting the fault state and entering the init state | Once all AIR shutdown faults are cleared and AIR- is open, the BMS must exit the fault state and enter the init state.
```

Comment:
@kirons96 hmmm wondering if we should also reset the AIR shd latch here as well

Reply:
nope, see my other comment about the latch. if there's a latched fault, turning the TSMS will do nothing. you'll have to push the button for the TSMS to close AIR-

Reply:
ok